src/metabase/analytics/prometheus.clj

```diff
@@ -331,7 +370,20 @@
   ([metric labels amount]
    (when-not system
      (setup!))
-   (prometheus/inc (:registry system) metric labels amount)))
+   (prometheus/inc (:registry system) metric (qualified-vals labels) amount)))
+
+(defn dec!
+  "Call iapetos.core/dec on the metric in the global registry.
+   Inits registry if it's not been initialized yet."
```

Comment:
Ditto, calling out that this is only for gauges could help people who don't know that counters are monotonic.